engine/src/flutter/display_list/dl_storage_unittests.cc

```diff
@@ -21,7 +21,7 @@ TEST(DisplayListStorage, Allocation) {
   EXPECT_NE(storage.allocate(10u), nullptr);
   EXPECT_NE(storage.base(), nullptr);
   EXPECT_EQ(storage.size(), 10u);
-  EXPECT_EQ(storage.capacity(), DisplayListStorage::kDLPageSize);
+  EXPECT_EQ(storage.capacity(), 8192u);
```

Comment:
Why the change from the expected constant?

Also, should be at least one test that tries to verify that it will go from N to N*2 instead of N+PageSize? Perhaps a loop that keeps doing a simple DrawRect until the size is larger than some PoT multiple of page size and then another loop that keeps doing DrawRect until it changes again and verifies that it changes to NPOT?

Reply:
Also NPOT is not external, but it would be nice to have some tests for it.